vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -150,6 +150,22 @@ paths:
         404:
           description: The VM instance could not be resized because it is not created.
 
+  /vm.resize-zone:
+    put:
+      summary: Resize a memory zone
+      requestBody:
+        description: The target size for the memory zone
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/VmResizeZone'
+        required: true
+      responses:
+        204:
+          description: The memory zone was successfully resized.
+        404:
```

Comment:
This must be wrong. Surely we generate a 500 in that case?

Reply:
Yes you're right.